HIP/hip-820.md

```diff
@@ -71,38 +71,34 @@ When a dApp requires only the signature from the controller (wallet), it can use
+### hedera_executeTransaction
+When a dApp only requires the services of the controller to act as a relay to the Hedera network for submitting an already signed transaction, it can use the `hedera_executeTransaction` method.  This method accepts a base64 encoded protobuf representation of the Hedera API [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) message as input.  The controller decodes the message, interprets the contents, and if valid and approved, transmits the message unaltered to the Hedera network.  The controller shall wait for the initial response from the Hedera node and return the resulting Node Precheck Code generated by the submission.
 
 The controller is allowed to retry submitting the signed transaction to the Hedera Network for certain types of error conditions; such as gRPC connection droppage or Hedera `BUSY` signals.  This specification does not obligate the controller to retry, but recommends the practice to help improve perceived user experience for consumer applications.
 
 #### Parameters
-`signedTransaction` – a base64 encoding of the protobuf endoded Hedera API [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) message.  The message must include the [bodyBytes](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L35C11-L35C11) field representing the transaction, and the [sigMap](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L40C13-L40C13) field containing one or more valid signatures.
+`signedTransaction` – a base64 encoding of the protobuf endoded Hedera API [`TransactionList`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/sdk/transaction_list.proto) message.  The message must include the [bodyBytes](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L35C11-L35C11) field representing the transaction, and the [sigMap](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L40C13-L40C13) field containing one or more valid signatures.
```

Comment:
Same reasons against this change as stated above.

Reply:
cool, safe to say we can arrive at the same conclusion as this thread - https://github.com/hashgraph/hedera-improvement-proposal/pull/839#discussion_r1414311878